gammapy/irf/psf_analytical.py

```diff
@@ -354,3 +358,46 @@ def info(self, fractions=[0.68, 0.95], energies=Quantity([1., 10.], 'TeV'),
                            "E = {2:4.1f}: {3:5.8f}\n"
                            "".format(100 * fraction, theta, energy, radius))
         return ss
+
+
+
+    def to_table_psf(self, theta=None, offset_max=None, binsz=0.01, exposure=None):
+        """
+        Convert triple Gaussian PSF ot table PSF.
+
+        Parameters
+        ----------
+        theta : `~astropy.coordinates.Angle`
+            Observation offset. Default theta = 0 deg.
+        offset_max : `~astropy.coordinates.Angle`
+            Maximum offset from PSF center. Default offset_max = 2.5 deg
+        binsz : float
+            Size of the spatial bins in deg / pixel. Default binsz = 0.01 deg
+        exposure : `~astropy.units.Quantity`
+            Energy dependent exposure. Should be in units equivalent to 'cm^2 s'.
+
+        Returns
+        -------
+        tabe_psf : `~gammapy.irf.EnergyDependentTablePSF`
+            Instance of `EnergyDependentTablePSF`.
+        """
+        # Convert energies to log center
+        ebounds = EnergyBounds.from_lower_and_upper_bounds(self.energy_lo, self.energy_hi)
+        energies = ebounds.log_centers
+        
+        # Default for exposure
+        exposure = exposure or Quantity(np.ones(len(energies)), 'cm^2 s')
+        theta = theta or Angle(0, 'deg')
+        offset_max = offset_max or Angle(2.5, 'deg')
+
+        offsets = Angle(np.arange(0, offset_max.degree, binsz), 'deg')
+
+        psf_value = Quantity(np.empty((len(energies), len(offsets))), 'deg^-2')
+
+        for i, energy in enumerate(energies):
+            psf_gauss = self.psf_at_energy_and_theta(energy, theta)
+            psf_value[i] = Quantity(psf_gauss(offsets, np.zeros_like(offsets)), 'deg^-2')
```

Comment:
Ok so let's forget the interpolation for now!

@cdeil 
ok for implementing the king profile.
I will have to develop the same kind of class as the ones in gauss.py in morphology ?
I wait for this PR to be finish. If you have some inputs before you leave in holiday of how I should implement this tell me :-)